config.yml

```diff
@@ -89,9 +93,9 @@ params:
           <br>
           <br>
           **[>> Sign-up for news <<](https://bit.ly/openfeature-signup)**
-    socialIcons:
-        - name: github
-          url: "https://github.com/open-feature"
+#    socialIcons:
```

Comment:
I would rather keep it

Reply:
Let's add it back in once we have other social channels. It seems out of place with only one.